src/components/InformationsBar/SummaryCard/styles.ts

```diff
@@ -2,6 +2,7 @@ import { Card } from '@mui/material';
 import styled from 'styled-components';
 
 export const Container = styled(Card)`
+  top: 150px;
   position: fixed;
   z-index: 2;
```

Comment:
Try to add this z-index on `InformationsBar` styles, should work well there too, so you can remove this from here and here https://github.com/gpmc-lab-ufrgs/atlas/pull/34/files?diff=unified&w=0#:~:text=position%3A%20fixed%3B-,z%2Dindex%3A%202%3B,-font%2Dsize%3A%2014px

Reply:
Done